### PR TITLE
Console test

### PR DESCRIPTION
**Pull request by alain-rs**
> ### Motivate of the pull request
> - [ x] To address an existing issue. If so, please provide a link to the issue: <issue id>
> - [ ] Breaking new feature. If so, please describe details in the description part.

> ### Describe the technical details
> #### What is currently done? (Provide issue link if applicable)
> 
> 
> 
> 
> 
>
> #### What does this pull request change?
> 
> 
> 
> 
> 

> ### Which part of the code base require a change
> 
> - [ ] Library: <Specify the library name>
> - [ ] Plug-in: <Specify the plugin name>
> - [ ] Engine
> - [ ] Documentation
> - [ ] Regression tests
> - [ ] Continous Integration (CI) scripts

> ### Impact of the pull request

> - [ ] Require a change on Quality of Results (QoR)
> - [ ] Break back-compatibility. If so, please list who may be influenced.

Console test fails if error during test

### CODE REVIEW

**Comment by KochynVolodymyr** [comment]
This changes doesn't solve the problem. If **console_proc** return TCL_ERROR - test still pass.

File: src/Console/Test/console_main.cpp [hunk 1]
```diff
@@ -51,14 +51,12 @@ void registerExampleCommands(FOEDAG::Session* session) {
...
-    QString fileName{"gui_console_proc.tcl"};
-    if (!path.exists(fileName)) return TCL_ERROR;
-    QString fullPath = path.filePath(fileName);
-    FOEDAG::TclConsoleWidget* console = FOEDAG::Init(interp);
-    QString res = console->getPrompt() + "source " + fullPath +
-                  "\nHello world\n" + console->getPrompt();
-    CHECK_EXPECTED("source " + fullPath, res)
+    QFile file(argv[1]);
+    QString expected(argv[2]);
+    if (!file.exists()) return TCL_ERROR;
```
Comment (KochynVolodymyr) [inline src/Console/Test/console_main.cpp:56]: if file not exists test will pass successfully.

Comment (alain-rs) [reply src/Console/Test/console_main.cpp:56]: exit (1) in that case. Please add it.
I checked that if the Tcl code fails or there is a diff, the test fails and the CI fails.

Comment (KochynVolodymyr) [reply src/Console/Test/console_main.cpp:56]: it still not solve the problem for the whole project. Everyone will expect that **return TCL_ERROR** will cause the CI fail. But it is not. That was the idea of the issue.

**Pull request merged**